include/wabt/interp/interp.h

```diff
@@ -589,11 +588,6 @@ struct Value {
     assert(t == type || type == ValueType::Any);
   }
   ValueType type;
-#else
-  Value() : v128_(0, 0, 0, 0) {}
-  void SetType(ValueType) {}
-  void CheckType(ValueType) const {}
-#endif
```

Comment:
I believe you can leave SetType/CheckType inside the `ifdef`.  The important thing is that the sizeof the struct be the same in both debug and release I think,

Reply:
@sbc100 Correct.